fleet/s/dev/c/ruka/observability/fleet.yaml

```diff
@@ -0,0 +1,14 @@
+defaultNamespace: monitoring
+helmCharts:
```

Comment:
This is not a valid key in a `fleet.yaml` file. https://fleet.rancher.io/ref-fleet-yaml